### PR TITLE
adds homework and updated weekly overview

### PR DESCRIPTION
**Pull request by shanghaiellen**
This adds the homework and Ellen's time on Friday. It also adds an updated weekly overview for week 18.

### CODE REVIEW

File: week17/friday/AngularJS-intro.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+    function init() {
+      $('.button').click(function() {
+        alert('Hi!');
+      });
+    }
+    window.onload = init;
+
+In AngularJS, on the other hand, all of this logic is moved to the view and 
+the controller. If you want to add a click event to a div in AngularJS, you
```
Comment (davidmfoley) [inline week17/friday/AngularJS-intro.md]: perhaps a note about the different definition of "controller" in angular as compared with "controller" in rails?

File: week17/friday/AngularJS-intro.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+`<button ng-click="alert('Success!')">Did it work?</button>`
+
+`ng-click` runs the code to the right of the equal sign when that HTML element is
+clicked. Sometimes, this is evaluated entirely in the view, like in this instance;
+often, it is a custom method that you define in your controller. In this case, when
+you click on the button, you see an alert.
+
+####Custom directives
+
+In addition to the standard directives that come with AngularJS, you also have the
```
Comment (davidmfoley) [inline week17/friday/AngularJS-intro.md]: Maybe a note about usage of directives (as "custom html elements/attributes") or something?

File: week18/cookbook-project.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    - Description
+    - String/text of directions
+- Recipes have many Ingredients
+- Ingredients have many Recipes
+- Recipes can be viewed as a list (an index page)
+    - This list can be filtered by a key word on the front-end
+- Recipes can be viewed individually (a show page)
+- New recipes can be created
+    - These recipes are saved to the database
+- All data goes through the Express/Node.js data source
```
Comment (davidmfoley) [inline week18/cookbook-project.md]: rad :+1:

File: week17/friday/AngularJS-intro.md (hunk 2 above)
Comment (shanghaiellen) [inline week17/friday/AngularJS-intro.md]: I talk about that some earlier in this section, at around line 138. Is that what you mean?

"Just like you can add an href to an anchor tag which your HTML 
then turns into a link, which looks like `<a href="foo">`, you can add 
ng- directives to your Angular app which will then run the appropriate logic. 
You add these to your HTML just like you add regular HTML attributes" ....etc

File: week17/friday/AngularJS-intro.md (hunk 1 above)
Comment (shanghaiellen) [inline week17/friday/AngularJS-intro.md]: I added a short note about that. We'll cover it more thoroughly next week. Thanks for the feedback!

**Pull request merged**